### ISSUE TITLE
Add captions to video on the homepage

### ISSUE THREAD

**Issue by warpech** [Starcounter/starcounter.io#19]
As @per-samuelsson pointed out, it is not very clear what is the purpose of the homepage video. It should be redone with text captions (I have ScreenFlow to do it) and voiceover or music.

**Comment by warpech** [comment]
Remove "using System" where it's not used
